### ISSUE TITLE
Resources - Acquisitions can't handle negative values 

### ISSUE THREAD

**Issue by jzairo** [coral-erm/coral#645]
In the resources module and acquisitions tab for a resource, it is not possible to add a negative value to apply a refund or credit to a particular resource. It would be helpful to add this when a library receives a credit or refund from a vendor. 

A current workaround is going to Cost Details in administration and creating a credit and refund option. This way when you export your resources you can see an entry for a credit or refund.

**Comment by lmortenson1** [comment]
I agree that being able to input negative values would be helpful. Is this something that's being considered yet?
